lua/astrocommunity/pack/ansible/init.lua

```diff
@@ -0,0 +1,24 @@
+local utils = require "astronvim.utils"
+
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "yaml")
+      end
+    end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "ansible-lint") end,
```

Comment:
you want the `null-ls` name, so it is automatically registered https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/doc/BUILTINS.md#ansiblelint

Reply:
and add a mapping here: https://github.com/jay-babu/mason-null-ls.nvim/blob/73c68abdf65279e41526eb152876511a8ae84ea2/lua/mason-null-ls/mappings/source.lua#L13